### PR TITLE
Added a transition to the buttons Aasanas on Yoga Therapy Page

### PR DESCRIPTION
**Pull request by MohitNamdev22**
Issue #349 
I've given the transition to the buttons on hover on Aasanas Section.

image

### CODE REVIEW

**Comment by Susmita-Dey** [comment]
@MohitNamdev22 have you pulled the recent changes? 
Seems like you haven't. The site has gone some changes after merging some PRs.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Comment by MohitNamdev22** [comment]
@Susmita-Dey I had pulled the code yesterday before starting working on this issue

**Comment by Susmita-Dey** [comment]
@MohitNamdev22 Your changes will override the PR changes that was merged recently. Kindly try to fetch and pull. I'm saying this because laughing therapy has no coming soon page now as in the original site which is not reflected in your PR.

**Comment by MohitNamdev22** [comment]
Okay no issues, I'll pull it again today, then merge it 👍

**Comment by MohitNamdev22** [comment]
@Susmita-Dey Pulled & merged the latest code to my repo

File: css/yoga.css [hunk 1]
```diff
@@ -219,6 +219,7 @@ header {
   color: green;
   font-weight: bolder;
   border: 2px solid green;
+  transition: 0.3s ease-in-out;
```
Comment (AliReza1083) [inline css/yoga.css:222]: If you want to code like a professional, there is no need like this:
```diff
- transition: 0.3s ease-in-out;
+ transition: .3s ease-in-out;
```

Overall looking Great.

**Review by AliReza1083 — COMMENTED** [review]
I reviewed You code, and I came up with this.
Wish You all the best.

File: css/yoga.css (hunk 1 above)
Comment (MohitNamdev22) [reply css/yoga.css:222]: Sure Ali, I'll keep that in mind

**Comment by Susmita-Dey** [comment]
@MohitNamdev22 Can you pull the recent changes once more?
After you're done kindly tag me here and also on discord to merge your PR.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by MohitNamdev22** [comment]
Okay @Susmita-Dey 👍

**Comment by MohitNamdev22** [comment]
@Susmita-Dey I've pulled the latest code

**Review by Susmita-Dey — APPROVED** [review]
LGTM. 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**